packages/xstate-store/src/types.ts

```diff
@@ -340,3 +346,18 @@ export type Selector<TContext, TSelected> = (context: TContext) => TSelected;
 export interface Selection<TSelected> extends Subscribable<TSelected> {
   get: () => TSelected;
 }
```

Comment:
This is basically redundant now but to avoid breaking the external types you can keep it as an alias:
```suggestion
export type Selection<TSelected> = Readable<TSelected>;
```